docs/en/05_Releases_and_changelogs/cwp_2.6.0.md

```diff
@@ -0,0 +1,23 @@
+# 2.6.0 (Unreleased)
+
+## Overview
+
+This release includes CMS and Framework version 4.6.0.
+
+ * [Framework 4.6.0](https://docs.silverstripe.org/en/4/changelogs/4.6.0/)
+
+
+Upgrading to Recipe 2.6.0 is recommended for all CWP sites. This upgrade can be carried out by any development team familiar with SilverStripe. However, if you would like SilverStripe's assistance, you can request support via the [Service Desk](https://www.cwp.govt.nz/service-desk/new-request/).
+
+
+### Multi-factor authentication (MFA) modules become a part of the default CWP installation
+
+The following modules are now included into [cwp/installer](https://github.com/silverstripe/cwp-installer)
```

Comment:
@dnsl48 Can we please mention here that WebAuthn is available as an optional module that you should consider installing if your users are likely to have hardware tokens? And link to more info about those modules, so that both website owners and devs can make informed choices. Potentially userhelp and dev docs?

Reply:
We'll update it as part of https://github.com/silverstripe/cwp/issues/269